.eslintrc.js

```diff
@@ -15,8 +15,8 @@ const LICENSE_HEADER = `
  */
 `
 
-module.exports = {	
-  root: true,	
+module.exports = {
```

Comment:
Were the changes to this file added to address the issue seen here? https://github.com/opensearch-project/security-dashboards-plugin/pull/1649#issuecomment-1810284144

There was a PR recently merged into main that resolves the babel import error: https://github.com/opensearch-project/security-dashboards-plugin/pull/1652